app/controllers/application_controller.rb

```diff
@@ -365,7 +365,7 @@ def get_page_title(fandom, author, title, options = {})
 
     @page_title = ""
     if logged_in? && !current_user.preference.try(:work_title_format).blank?
-      @page_title = current_user.preference.work_title_format
+      @page_title = current_user.preference.work_title_format.dup
```

Comment:
Yeah. I don't remember the specifics, but there's a cucumber test that involves viewing a work, and then accessing `User.current_user` and changing the preferences. This causes the altered `work_title_format` to be saved as well.

I changed it here rather than calling `reload` in the tests because if there ever ends up being a call to `preference.save` in the `WorksController` (unlikely to happen, but the `WorksController` does have the adult content check), it'd be good to avoid accidentally clobbering the work title format.